debian/cortx-motr.install

```diff
@@ -30,11 +30,11 @@ usr/lib/*/cortx-motr/motr-server
 usr/lib/*/cortx-motr/motr-dixinit
 usr/lib/udev/rules.d/99-motr.rules
 usr/sbin/m0gendisks
-usr/sbin/m0beck
+# usr/sbin/m0beck
 usr/sbin/m0protocol
 usr/sbin/m0ham
 usr/sbin/m0console
-usr/sbin/m0genfacts
+# usr/sbin/m0genfacts
```

Comment:
Why are these two binary files not generated and missing on debian?

This might be bug in original building configuration.

Reply:
I know the reason: this Jira ticket CORTX-32593

It was disabled by this commit: 3571bc94978f56264f0c9a41d0fc7a637e9c747f